SPECS/qemu/qemu.spec

```diff
@@ -1276,7 +1263,7 @@ mkdir -p %{qemu_kvm_build}
 
 run_configure() {
     ../configure  \
-        --cc=%{__cc} \
+        --cc=gcc \
```

Comment:
I'd actually revert this whole commit except for maybe this one line here. The linter is not great at understanding everything and now `Requires` are mixed in with `BuildRequires`. Similarly, the `%define requires_*` lines were moved away from other ones, so they are now harder to find.